app/poc-common/clinical-services/directives/clinicalServices.js

```diff
@@ -49,23 +51,23 @@
         patientService.getPatient(vm.patientUuid)
       ])
         .then(function (result) {
-          var todaysVisit = result[0];
+          vm.patientCheckedIn = !!result[0];
```

Comment:
This is supposed to be `!result[0]` right?

Reply:
```!!result[0]``` is correct, even better is ```result[0] !== null```. :see_no_evil:

Reply:
Just learned something new :grin: